### ISSUE TITLE
Default memory options are still added even if they are present in sbtopts

### ISSUE THREAD

**Issue by netvl** [sbt/sbt-launcher-package#81]
According to `sbt-launch-lib.bash`, default memory options are not supposed to be added if the user specified memory options themselves:
```
get_mem_opts () {
# if we detect any of these settings in ${java_opts} we need to NOT output our settings.
# The reason is the Xms/Xmx, if they don't line up, cause errors.
```
However, it fails to detect memory settings if they are specified in sbtopts file. For example, my `/usr/local/etc/sbtopts` contains these lines:
```
% cat /usr/local/etc/sbtopts | sed '/^\#/d' | grep -v '^$'
-sbt-create
-java-home "$(/usr/libexec/java_home -v1.7)"
-J-Xmx4096M
-J-XX:MaxPermSize=2G
-J-XX:ReservedCodeCacheSize=1G
-J-XX:-OmitStackTraceInFastThrow
-J-XX:+CMSClassUnloadingEnabled
-J-XX:+UseConcMarkSweepGC
```
But if I launch `sbt -v` I get this:
```
% sbt -v
[process_args] java_version = '1.7.0_60'
# Executing command line:
/Library/Java/JavaVirtualMachines/jdk1.7.0_60.jdk/Contents/Home/bin/java
-Xms1024m
-Xmx1024m
-XX:ReservedCodeCacheSize=128m
-XX:MaxPermSize=256m
-Xmx4096M
-XX:MaxPermSize=2G
-XX:ReservedCodeCacheSize=1G
-XX:-OmitStackTraceInFastThrow
-XX:+CMSClassUnloadingEnabled
-XX:+UseConcMarkSweepGC
-jar
/usr/local/Cellar/sbt/0.13.6/libexec/sbt-launch.jar
```
That is, default options are still added.

**Comment by mbauhardt** [comment]
I run into the same. Any news on that?

**Comment by jsuereth** [comment]
No, I'm going to try to create a test-suite that can check script arg passing via -debug.

**Comment by frosforever** [comment]
:+1:

**Comment by ddugovic** [comment]
Workaround (fix?): I added to my `sbt-launch-lib.bash` the following (near the start of `get_mem_opts`):

    elif [[ "${SBT_OPTS}" == *-Xmx* ]] || [[ "${SBT_OPTS}" == *-Xms* ]] || [[ "${SBT_OPTS}" == *-XX:MaxPermSize* ]] || [[ "${SBT_OPTS}" == *-XX:MaxMetaspaceSize* ]] || [[ "${SBT_OPTS}" == *-XX:ReservedCodeCacheSize* ]]; then
         echo ""

**Comment by ddugovic** [comment]
Sorry for my slow response.  I've tested my `sbt-launch-lib.bash` change and now understand that it is the cleanest way to fix this issue.  I'll commit and PR my fix.

**Comment by eed3si9n** [comment]
Should be fixed by #101
